src/scripts/components/surah/Line.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable consistent-return */
+
+import React from 'react';
+import * as AudioplayerActions from 'actions/AudioplayerActions';
+import ReactZeroClipboard from 'react-zeroclipboard';
+import debug from 'utils/Debug';
+
+class Line extends React.Component {
+  //componentDidMount() {
+  //  console.log( 'im line', window );
+  //}
+
+  text() {
+    if (!this.props.line[0].char) { // TODO shouldn't be possible, remove this clause
+      return;
+    }
+
+    let text = this.props.line.map(data => {
```

Comment:
'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'arrow function syntax (=>)' is only available in ES6 (use esnext option).